projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageBuilder.java

```diff
@@ -253,14 +256,26 @@ public boolean isBuildingJobInProgress(String dataJobName) {
   private Map<String, String> getBuildParameters(DataJob dataJob, JobDeployment jobDeployment) {
     String jobName = dataJob.getName();
     String jobVersion = jobDeployment.getGitCommitSha();
+    String baseImage = null;
```

Comment:
```suggestion
    String dataJobbaseImage = null;
```
How about that or something similar?

Reply:
Changed, thanks